install-cockroachdb.md

```diff
@@ -378,18 +378,15 @@ $(document).ready(function(){
 
     <div class="highlighter-rouge"><pre class="highlight"><code>$ sudo docker version</code></pre>
     </div>
-    <p>If you don't see the server listed, run:</p>
-
-    <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="linux-docker-step2"><span class="gp" data-eventcategory="linux-docker-step2">$ </span>sudo docker -d &amp;</code></pre>
-    </div>
-
-    <div class="bs-callout bs-callout-info"> On Linux, Docker needs sudo privileges.</div>
+    <p>If you don't see the server listed, start the Docker daemon. </p>
   </li>
   <li>
     <p>Pull the official CockroachDB image from <a href="https://hub.docker.com/r/cockroachdb/cockroach/" data-eventcategory="linux-docker-step3">Docker Hub</a>:</p>
 
     <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="linux-docker-step3"><span class="gp" data-eventcategory="linux-docker-step3">$ </span>sudo docker pull cockroachdb/cockroach:{{site.data.strings.version}}</code></pre>
     </div>
+
+    <div class="bs-callout bs-callout-info"> On Linux, Docker needs sudo privileges.</div>
```

Comment:
I'd remove this - it looks like it applies just to this bullet but really you're also calling `sudo docker version` above.